embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -35,4 +41,9 @@ internal sealed class EmbType {
  */
 internal interface TelemetryType {
     val description: String
+
+    /**
+     * Return the key name used by this attribute when is used inside of OpenTelemetry objects
```

Comment:
I'm not 100% on how this would be used. Does an `EmbraceAttribute` call this, and why is the value prefixed with `type`? An example is probably all I need to grok this

Reply:
The attributeName is the key name used in the attributes map. The next PR should clarify this - eventually I'll merge this in with the `EmbraceAttribute` stuff and I think it'll make more sense.